Sources/MongoSwift/MongoCollection.swift

```diff
@@ -12,6 +12,21 @@ public struct DropCollectionOptions: Codable {
     }
 }
 
+/// Options to use when renaming a collection.
+public struct RenamedCollectionOptions: Codable {
+    /// Specifies a collation.
+    public var collation: BSONDocument?
```

Comment:
I don't believe the `renameCollection` command supports a collation. (https://docs.mongodb.com/manual/reference/collation/#operations-that-support-collation)

Reply:
That's right, collation is not an option expected by the server. https://jira.mongodb.org/browse/CDRIVER-3819
The C Driver will update their documentation.